### PR TITLE
Override `assignBuildNumber` rather than `saveNextBuildNumber`

### PR DESCRIPTION
**Pull request by jglick**
Avoids test failures due to (ab)use of EasyMock when running against https://github.com/jenkinsci/jenkins/pull/9846.

I would appreciate a release after merge, to prevent problems in PCT.

### CODE REVIEW

File: src/test/java/hudson/plugins/jacococoveragecolumn/AbstractJaCoCoCoverageColumnTest.java [hunk 1]
```diff
@@ -89,9 +89,10 @@ public MyRun getLastSuccessfulBuild() {
 				}
 			}
 
-			@Override
-			protected synchronized void saveNextBuildNumber() {
-			}
+            @Override
```
Comment (jglick) [inline src/test/java/hudson/plugins/jacococoveragecolumn/AbstractJaCoCoCoverageColumnTest.java:92]: Ignore WS (tabs).

**Comment by jglick** [comment]
([CloudBees-internal issue](https://cloudbees.atlassian.net/browse/BEE-52532))

**Comment by jglick** [comment]
@centic9 can you please release this? https://github.com/jenkinsci/bom/pull/3912#issuecomment-2458266836

(BTW I recommend [setting up CD](https://www.jenkins.io/doc/developer/publishing/releasing-cd/) to make this sort of thing be less of a personal effort.)

**Comment by centic9** [comment]
Did a release now, but I also orphaned the plugin now, hopefully someone else can take over.

**Comment by uhafner** [comment]
> Did a release now, but I also orphaned the plugin now, hopefully someone else can take over.

@centic9 I wonder if it wouldn't be better to rather mark the plugin as deprecated: its functionality is fully covered by the https://github.com/jenkinsci/coverage-plugin.

**Pull request merged**